eeps/eep-0007.md

```diff
@@ -392,12 +393,13 @@ This BIF accepts the following parameters:
 
 - **Port**
 
-    A port opened towards the required driver/library (that
-    **must** have been loaded with `erl_ddll:load_library/3`).
+  A port opened towards the required driver/library (that
+  **must** have been loaded with `erl_ddll:load_library/3`).
 
 - **OptimizedCall**
-    A tuple with the function index (i.e. its position in
-    the preload list) followed by its arguments (if any).
+
```

Comment:
Without white lines after this, options (there are many of these instances) are followed by text, in the same line (in the generated HTML).